.github/workflows/publish.yml

```diff
@@ -37,7 +37,6 @@ jobs:
   deploy-prod:
     name: Publish ODR & Sync STAC
     runs-on: ubuntu-latest
-    concurrency: publish-${{ github.ref }}
     needs: main
```

Comment:
Will this job run if the `main` step is skipped?

Reply:
This isn't any different from how it was originally.

Reply:
I think we got lucky here, the `main` could be skipped if a newer one is running, but due to the merge queue taking about the same length of time as the `main` there were no skips.

if the merge queue is ever faster than the validate step it will likely skip this entire step?

Reply:
>No, because I've removed the concurrency.

You did this after I made this comment.. but yes removing concurrency limit from `main` should hopefully ensure this step is not skipped.

Reply:
No, I did not. The change was there in the first pushed commit. But I take it that means this issue is resolved?

Reply:
https://github.com/linz/imagery/pull/394/commits/79b8030028767bcfd13c22c547574f07b9258f22 is the first commit, no removal of `main` concurrency

https://github.com/linz/imagery/pull/394/commits/e0b151e75e8c8b36b29f28cb157f0c473ef6c612 was committed post my comments with the removal of `main` concurrency

Reply:
Yes, you're right.